### PR TITLE
Improve Summaries

### PR DESCRIPTION
**Pull request by eksperimental**
and other fixes

### CODE REVIEW

File: lib/mix/lib/mix/scm.ex [hunk 1]
```diff
@@ -1,7 +1,7 @@
 defmodule Mix.SCM do
   @moduledoc """
   This module provides helper functions and defines the
-  behaviour required by any SCM used by mix.
+  behaviour required by any SCM used by `Mix`.
```
Comment (josevalim) [inline lib/mix/lib/mix/scm.ex]: No backticks I think. We are referring to Mix as the tool, right? And not the module `Mix`.

File: lib/elixir/lib/module.ex [hunk 2]
```diff
@@ -1,8 +1,8 @@
 defmodule Module do
   @moduledoc ~S'''
-  This module provides many functions to deal with modules during
-  compilation time. It allows a developer to dynamically attach
-  documentation, add, delete and register attributes and so forth.
+  Provides functions to deal with modules during compilation time.
+  It allows a developer to dynamically add, delete and register
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Let's break this into its own paragraph.

File: lib/elixir/lib/list.ex [hunk 3]
```diff
@@ -1,8 +1,8 @@
 defmodule List do
   @moduledoc """
   Implements functions that only make sense for lists
```
Comment (josevalim) [inline lib/elixir/lib/list.ex]: Let's shorten this one too. Maybe: "Specialized functions that works only on lists". The rest can be its own paragraph.

File: lib/elixir/lib/keyword.ex [hunk 4]
```diff
@@ -1,5 +1,7 @@
 defmodule Keyword do
   @moduledoc """
+  A set of functions for working with keywords.
```
Comment (josevalim) [inline lib/elixir/lib/keyword.ex]: :+1:

File: lib/elixir/lib/kernel/special_forms.ex [hunk 5]
```diff
@@ -1,12 +1,11 @@
 defmodule Kernel.SpecialForms do
   @moduledoc """
-  In this module we define Elixir special forms. Special forms
-  cannot be overridden by the developer and are the basic
+  Special forms cannot be overridden by the developer and are the basic
```
Comment (josevalim) [inline lib/elixir/lib/kernel/special_forms.ex]: I would invert the order of the sentence "Special forms are the basic building blocks of Elixir and ..."

File: lib/elixir/lib/kernel.ex [hunk 6]
```diff
@@ -7,11 +7,12 @@ import :elixir_bootstrap
 
 defmodule Kernel do
   @moduledoc """
-  `Kernel` provides the default macros and functions
-  Elixir imports into your environment. These macros and functions
-  can be skipped or cherry-picked via the `import` macro. For
-  instance, if you want to tell Elixir not to import the `if`
-  macro, you can do:
+  Provides the default macros and functions Elixir imports into your
```
Comment (josevalim) [inline lib/elixir/lib/kernel.ex]: :+1:

File: lib/mix/lib/mix/scm.ex (hunk 1 above)
Comment (eksperimental) [inline lib/mix/lib/mix/scm.ex]: correct.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**